src/anujaD/Bank.java

```diff
@@ -0,0 +1,102 @@
+/*Assignment - 13 : 1st Sep'2022
+
+Create a Banking System which has the following functionality.
+
+Create two objects to perform the below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times the individual debit, credit & printBalance method is called .
+Hint : Create 3 non static instance variables.
+
+when calling the method on the first object.
+Output: user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+
+when calling the method on the second object.
+Output: user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variables will be required.
+Output:
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time
+
+NOTE : You should have setUserDetails method to set username and initial Balance .*/
+
+package anujaD;
+
+public class Bank {
+	int creditamnt =0;
+	int debitamnt=0;
+	int currentbalance=0;
+	int balance;
+	
+	static int Ind_Debit_amt;
+	static int Ind_Credit_amt;
+	static int Ind_printbal_amt;
+
+	void credit(int num) {
+		balance = (balance+num);
+		creditamnt++;
+		Ind_Credit_amt++;
+	}
+	void debit(int num) {
+		balance= (balance-num);
```

Comment:
before debit, we should check for the condition that we have sufficient balance or not